src/pep8ext_naming.py

```diff
@@ -291,6 +291,25 @@ class ClassNameCheck(BaseASTCheck):
     Classes for internal use have a leading underscore in addition.
     """
     N801 = "class name '{name}' should use CapWords convention"
+    N818 = "class name '{name}' should use Error suffix as it is an Exception"
```

Comment:
Maybe `"exception class '{name}' should be named with an Error suffix"`?

I think we want to make this message very obvious and actionable.